.gitignore

```diff
@@ -4,3 +4,9 @@ coverage
 dist
 .opt-in
 .opt-out
+
+### JetBrains template
+# Covers JetBrains IDEs: IntelliJ, RubyMine, PhpStorm, AppCode, PyCharm, CLion, Android Studio and Webstorm
+# Reference: https://intellij-support.jetbrains.com/hc/en-us/articles/206544839
+
+.idea/*
```

Comment:
This should be in your global gitignore file: https://help.github.com/articles/ignoring-files/#create-a-global-gitignore

Do you mind removing it from this PR?